build/charts/flow-aggregator/templates/deployment.yaml

```diff
@@ -79,6 +79,10 @@ spec:
           readOnly: true
         - mountPath: /var/log/antrea/flow-aggregator
           name: host-var-log-antrea-flow-aggregator
+        {{- if and .Values.clickHouse.tls.customCACert .Values.clickHouse.tls.customCACert }}
+        - name: clickhouse-ca
+          mountPath: /etc/ssl/certs
```

Comment:
is using this directory really necessary in our case? We are explicitly providing the CA certificate when we create the ClickHouse client connection, so I think using this standard unix directory can only create confusion. I'd recommend just mounting the certificate to `/etc/flow-aggregator`.

Reply:
We already use `/etc/flow-aggregator` for flow-aggregator conifg. I think we are not able to mount two volumes to the same path?
I changed the path to `/etc/flow-aggregator/ssl/certs`

Reply:
We already use `/etc/flow-aggregator` for flow-aggregator conifg. I think we are not able to mount two volumes to the same path?
I changed the path to `/etc/flow-aggregator/ssl/certs`

Reply:
You could use `subPath` and have both files in `/etc/flow-aggregator`.
https://stackoverflow.com/a/68179095/4538702

However, I am fine with using a subdirectory if it works. Maybe rename `/etc/flow-aggregator/ssl/certs` to `/etc/flow-aggregator/certs`

Reply:
I've renamed it to ` /etc/flow-aggregator/certs`. Thanks.